modules/overriding-default-node-ip-selection-logic.adoc

```diff
@@ -0,0 +1,77 @@
+// This is included in the following assemblies:
+//
+// ipi-install-configuration-files.adoc
+
+:_content-type: PROCEDURE
+[id="proc-overriding-default-node-ip-selection-logic_{context}"]
+= Optional: Overriding the default node IP selection logic 
+
+You can create a hint file that includes the `NODEIP_HINT` variable to override the default IP selection logic. This allows users to select a specific node IP address from the subnet `NODEIP_HINT` variable and specify which IP address is used. For example, if a node has two interfaces, `eth0` and `eth1`, with the IP addresses `10.0.0.10/24` and `192.0.2.5/25` respectively, and the default route points to `eth0`, the node IP address normally uses the `10.0.0.10` address. With `NODEIP_HINT`, you can specify that the node IP address use `192.0.2.5` by configuring `NODEIP_HINT` to point to `192.0.2.1`. 
+
+Use the following procedure to override the default node IP selection logic. 
+
+.Procedure
+
+. Add a hint file to your your `/etc/default/nodeip-configuration` file. The contents of your hint file must match the following example: 
++
+[source,text]
+----
+NODEIP_HINT=<node_ip_address>
+----
++
+[IMPORTANT]
+====
+* Do not use the exact IP address of a node as a hint, because `runtimecfg` treats it as a VIP and ignores it when looking for a node IP address. This causes the node using the hint IP address to fail to configure correctly. For example, the desired node IP address is `192.0.2.5`, use `192.0.2.1`. 
+* The IP address in the hint file is only used to determine the correct subnet. It will not receive traffic as a result of appearing in the hint file. 
+====
+
+. Activate the hint by creating a machine config manifest for both `master` and `worker` roles before deploying the cluster: 
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+ labels:
+   machineconfiguration.openshift.io/role: master
+ name: 99-nodeip-hint-master
+spec:
+ config:
+   ignition:
+     version: 3.2.0
+   storage:
+     files:
+     - contents:
+         source: data:text/plain;charset=utf-8;base64,[base64-encoded content of the file] <1>
+       mode: 0644
+       overwrite: true
+       path: /etc/default/nodeip-configuration
+----
++
+<1> Replace `[base64-encoed content of the file`] with the  base64-encoded content of the `/etc/default/nodeip-configuration` file. 
```

Comment:
I've not seen it done this way; How about

```
<encoded_contents>
```

We use the angle brackets in our docs.